README.md

```diff
@@ -132,3 +132,48 @@ This will only submit the data and not validate.
 If you are working with large VCF files and find that validation takes a very long time, you can add the
 argument `--shallow` to the command, which will validate only the first 10,000 lines in each VCF. Note that running
 shallow validation will **not** be sufficient for actual submission.
+
+
+## Leveraging Nextflow to parallelize the validation process  
+
+When running natively (i.e. not using docker), eva-sub-cli will use nextflow to run all the validation steps. When no options are provided nextflow will run as many tasks as there are available CPU on the machine executing it. To modify how many tasks can start and how nextflow will process each one, you can provide a nextflow configuration file in several ways:
```

Comment:
```suggestion
When running natively (i.e. not using Docker), eva-sub-cli will use a workflow management system called Nextflow to run all the validation steps. When no options are provided, Nextflow will run as many tasks as there are available CPUs on the machine executing it. To modify how many tasks can start and how Nextflow will process each one, you can provide a Nextflow configuration file in several ways.
```
* I think Nextflow and Docker should be capitalized throughout...
* Somewhere we should say briefly what Nextflow is as many people probably won't know, either here or even earlier when we define the dependencies.